src/vs/code/electron-sandbox/issue/issueReporterService.ts

```diff
@@ -283,6 +283,7 @@ export class IssueReporter extends Disposable {
 			const data = await this.issueMainService.$getReporterStatus(extension.id, extension.name);
 			return data;
 		} catch (e) {
+			console.error(e);
```

Comment:
I think this is fine being `console.error(e)` since it just applies to the issue reporter process.